### PR TITLE
Change the way to kill ContextBroker

### PR DESCRIPTION
**Pull request by jcalderin**
Use better logger
Add configuration to set the pid file and contextbroker log

### CODE REVIEW

File: test/acceptance/integration/tools/general_utils.py [hunk 1]
```diff
@@ -291,7 +295,19 @@ def start_cb(parms):
...
     if world.cb_pid != '':
         stop_cb()
-    getattr(api, runner)('{bin_path} {parms} >> /tmp/cb_acceptance_test.log'.format(bin_path=config['bin_path'], parms=parms))
+    if config['log_path'] == '':
+        log_path = '/tmp/acceptance'
+    else:
+        log_path = config['log_path']
+    getattr(api, runner)('mkdir -p {log_path}'.format(log_path=log_path))
+    if config['pid_file'] == '':
+        pid_path = '/tmp/acceptance/contextBroker.pid'
```
Comment (fgalan) [inline test/acceptance/integration/tools/general_utils.py]: What would happend if two acceptance runs happens at same time? Maybe is safer to use mktemp for this (cleaning up the directory after that).

Same comment for log_path.

File: test/acceptance/integration/tools/general_utils.py [hunk 2]
```diff
@@ -291,7 +295,19 @@ def start_cb(parms):
...
+    else:
+        pid_path = config['pid_file']
+    getattr(api, runner)('mkdir -p {pid_dir}'.format(pid_dir=pid_path[:pid_path.rfind('/')]))
+    command = '{bin_path} {parms} -logDir {log_path} -pidpath {pid_path} >> /tmp/cb_acceptance_test.log'.\
```
Comment (fgalan) [inline test/acceptance/integration/tools/general_utils.py]: Same problem here. An overall grep for "/tmp" in the code could help to locate all cases.

File: test/acceptance/integration/tools/general_utils.py (hunk 1 above)
Comment (jcalderin) [inline test/acceptance/integration/tools/general_utils.py]: The directory are in the properties file, if any wants to execute the tests at the same time, they have to change the directories

NTC

File: test/acceptance/integration/tools/general_utils.py (hunk 2 above)
Comment (jcalderin) [inline test/acceptance/integration/tools/general_utils.py]: Same as before

NTC

**Comment by fgalan** [comment]
LGTM (passing the ball to @iariasleon for final LGTM + merge + delete branch)

**Comment by iariasleon** [comment]
LGTM

**Pull request merged**